index-handlers/src/main/java/no/sikt/nva/nvi/index/model/InstitutionPoints.java

```diff
@@ -0,0 +1,42 @@
+package no.sikt.nva.nvi.index.model;
+
+import com.fasterxml.jackson.annotation.JsonProperty;
+import com.fasterxml.jackson.annotation.JsonTypeInfo;
+import com.fasterxml.jackson.annotation.JsonTypeName;
+import com.fasterxml.jackson.databind.annotation.JsonSerialize;
+import java.math.BigDecimal;
+import java.net.URI;
+import java.util.List;
+
+@JsonSerialize
+@JsonTypeInfo(
+    use = JsonTypeInfo.Id.NAME,
+    property = "type")
+@JsonTypeName("InstitutionPoints")
+public record InstitutionPoints(URI institutionId,
+                                BigDecimal institutionPoints,
+                                @JsonProperty("breakdown") List<CreatorAffiliationPoints> creatorAffiliationPoints) {
```

Comment:
why not to use default jsonName?
breakdown?

Reply:
I used breakdown when planning this task with @brinxmat, but probably just as good as `creatorAffiliationPoints`. @brinxmat What do you think?

Reply:
The naming confused Kir, so choose a better one… creatorAffiliationPoints is likely good, yes.